app/models/manifest_uploader.rb

```diff
@@ -5,20 +5,24 @@
 class ManifestUploader
```

Comment:
I am wondering whether we need to change the name of this? To something like e.g. LabwaresCreator. We could also then set a flag in the request to say whether it is coming from a manifest or not. Then the event could be set based on that flag. Just a thought - probably not needed at the moment.

Reply:
Good idea, will just leave for now though to keep it moving forward